src/main/java/nl/novi/Les11SpringBootTechItEasyApplication/dtos/TelevisionDto.java

```diff
@@ -0,0 +1,47 @@
+package nl.novi.Les11SpringBootTechItEasyApplication.dtos;
+
+import nl.novi.Les11SpringBootTechItEasyApplication.models.Television;
+
+public class TelevisionDto {
+    private Long id;
+    private String type;
+    private String brand;
+    private String name;
+    private Double price;
+    private Double availableSize;
+    private Double refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    private Integer sold;
+
+    public static TelevisionDto fromTelevision(Television television){
```

Comment:
In de opdracht staat inderdaad dat je deze functie hier moet neerzetten, maar ik zou het in de service zetten omdat ik het eigenlijk ook logica vind. In hoofdstuk 7.4 op Edhub staat ook dit:
"Deze modelklassen zijn eenvoudige klassen met enkele attributen en met getters en setters voor de attributen. Dit worden ook wel POJO's genoemd:

Plain Old Java Objects. "

Reply:
Voordat ik jouw review heb gelezen, heb ik al vanochtend alles veranderd en naar Service verplaatst